src/Services/Gateways/BaseGateway.php

```diff
@@ -10,11 +10,30 @@
 
 abstract class BaseGateway extends HttpService
 {
+    const COUNTRY_BRAZIL   = 'Brazil';
+    const COUNTRY_COLOMBIA = 'Colombia';
+    const COUNTRY_MEXICO   = 'Mexico';
+    const COUNTRY_ARGENTINA = 'Argentina';
+
     /**
      * @var Exchange
      */
     protected $exchange;
 
+    public static $maxInstalmentCountry = [
```

Comment:
It's advised to write a method on [CreditCard class](https://github.com/ebanx/benjamin/blob/master/src/Services/Gateways/CreditCard.php) returning a constant which then can be overridden on its specializations for each country.
Instalments don't belong in the BaseGateway as not every gateway has instalments.